### ISSUE TITLE
Config parse error

### ISSUE THREAD

**Issue by bensleveritt** [kmkzt/favicons-generate#2]
Welp, might be a little early to the party (3 day old repo), but I have an issue already!

Looks like there might be a need for a config, but there's no mention of one AFAICT.

```
❯ favicons-generate -v
0.0.12

❯ favicons-generate -i ./logo.svg -o ./favicons
config parse error
```

Good on you for making this though, clearly exactly what I was looking for!

**Comment by bensleveritt** [comment]
Ah, a little delving in and seems you need a config in the present working directory.

It'd be good if the one in dev would be used by default and then overriden by the user if they chose to (with a flag, to make it clear that's what's happening).

**Comment by kmkzt** [comment]
Thank you for building the issue.
will fix.

**Comment by kmkzt** [comment]
@bensleveritt 
I fixed it with v0.0.13.
